src/Config.php

```diff
@@ -48,8 +48,8 @@ public function __construct($hostname, $conf = null, CacheItemPoolInterface $cac
             // If there is no cacheItemPool, use shared memory for
             // caching the configuration and channel pool.
             if (! extension_loaded('sysvshm')) {
-                throw \RuntimeException(
-                'sysvshm extension is required to use this ItemPool');
+                error_log('sysvshm extension is required to use this ItemPool');
+                return new \Grpc\DefaultCallInvoker();
```

Comment:
Since this is in the constructor, I don't believe returning the call invoker here does anything. Is the intention to follow something similar to [this line](https://github.com/GoogleCloudPlatform/grpc-gcp-php/blob/86b9b6e2be9c62dc875ef0d2e667f3a8a693228c/src/Config.php#L41)?